lib/utils/common-flags.ts

```diff
@@ -97,18 +96,7 @@ export const deviceType = flags.string({
 	required: true,
 });
 
-export const deviceTypeIgnored = {
-	...(isV14()
-		? {}
-		: {
-				type: flags.string({
-					description: 'ignored - no longer required',
-					char: 't',
-					required: false,
-					hidden: true,
-				}),
-		  }),
-};
+export const deviceTypeIgnored = {};
```

Comment:
Can we completely drop this, or is it used anywhere else?

Reply:
It's used but it's pointless to keep, just doing `...{}` pretty much, dropping